types/soundmanager2/index.d.ts

```diff
@@ -54,6 +54,11 @@ declare namespace soundmanager {
         duration?: number | null;
     }
 
+    interface SoundManagerAudioFormat {
+        type: string[];
+        required: boolean;
```

Comment:
Missing the optional `related` property:

```suggestion
        required: boolean;
        related?: string[];
```